data/remnant missions.txt

```diff
@@ -1331,6 +1331,26 @@ mission "Remnant Return the Samples Compatibility Patch"
 		fail
 
 
+# "This mission is intended to serve as a notification that one timer has elapsed and how long to wait until the missions start again."
+mission "Remnant: Return the Samples 3 Exploration Hint"
+	name "Return the Samples Timer Elapsed"
+	description "According to your commlink, Plume has reached a milestone in his research."
```

Comment:
I feel like referring to "your commlink" is weird, probably since it's such a rarely used term in game. (It's used in the FW intro and a few times in the Remnant missions and that's it.) I'd suggest a change, but this mission actually always declines, so no need for a `name` or `description`.

Reply:
The description is there so that when <first> <last> uses their magical powers of prophecy to discern the true meaning of things, they can find it. (That's the best way I can think of to describe <first> <last>'s special ability, anyway. (along with their alternate ability - being able to rewind the past)

Reply:
Ain't that what we have comments for? 🤔

Reply:
Well, comments are data outside the framework. Exceptions. If we have a built in way to include the same info in a format the game, players, and things like mission builders will all handle in a uniform, predictable, way. Why not use it?

Reply:
I'm still not sure why you'd need `name` and `description` for a mission where they're never visible in game.